packages/headless/src/features/facets/range-facets/generic/range-facet-analytics-actions.ts

```diff
@@ -1,4 +1,4 @@
-import {FacetRangeMetadata} from 'coveo.analytics/src/searchPage/searchPageEvents';
+import type {FacetRangeMetadata} from 'coveo.analytics/dist/definitions/searchPage/searchPageEvents';
```

Comment:
Could the best fix be to export the types from `coveo.analytics` and then import them here as:

```
import {FacetRangeMetadata} from 'coveo.analytics';
```

Reply:
It'd be the best fix, I 💯 agree.

Given that imports were already done against `coveo.analytics/dist/definitions` in the project, I decided to address only the "we're importing .ts files and that's bad" part of the problem in this PR/Jira.

I estimated that we're not "contracting more debt" like this (but yeah, nor are we paying more).

Given all this and also that my underlying goal here is to address https://github.com/coveo/cli/issues/531, I preferred to not get on an odyssey with a 3rd repo to fix some small issues. 😅